### PR TITLE
Chapter 3

### PR DESCRIPTION
**Pull request by meimei0111**
ready

### CODE REVIEW

**Comment by meimei0111** [comment]
ready

**Comment by manhnt-0960** [comment]
confilcted

**Comment by manhnt-0960** [comment]
conflicted
please fix all conflict before "ready"

**Comment by meimei0111** [comment]
ready

**Comment by meimei0111** [comment]
I don`t know how to rebase the extra commits after solving the conflicts.

**Review by saobv-2223 — COMMENTED** [review]
Your PR is having 3 commits, merge to 1 commit please

**Comment by meimei0111** [comment]
Step 1: git branch -D master
Step 2: git fecth origin master:master
Step 3: git rebase master

(After that, it will become two joined line, but no effect)
Step 4: git push origin orign/
Step 5: git push origin chapter_n -f

I tried this but until step 5, it came out this
meimei@meimei-X556UF:~/Desktop/sample_app$ git push origin chapter_n -f
error: src refspec chapter_n does not match any.
error: failed to push some refs to 'https://github.com/meimei0111/sample_app.git'

**Comment by meimei0111** [comment]
ready

**Comment by meimei0111** [comment]
I would like to ask about, there still have 2 commits 
How should I do to solve the problem.

**Comment by ngocvt-0484** [comment]
please run
git reset --soft origin/master
git commit -m "chapter 3"
git push origin -f

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/static_pages.scss]: remove empty file

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ngocvt-0484) [inline app/helpers/static_pages_helper.rb]: remove empty file you not used

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<% provide :title, t(".home_title") %>
+<h1><%= t ".heading" %></h1>
+
+<p>
+	<%= t ".description_title" %>
```
Comment (ngocvt-0484) [inline app/views/static_pages/home.html.erb]: format your html
please tab 2 space

File: config/settings.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+links:
+  rails_tutorial:
+    home_page: http://www.railstutorial.org/
+    help_page: http://www.railstutorial.org/help
+    help_book: http://www.railstutorial.org/book
+    contact: https://www.railstutorial.org/contact    
+    news: https://news.learnenough.com
+    author: https://www.michaelhartl.com
+    screencast: https://www.learnenough.com/courses
+    ruby_on_rails: https://rubyonrails.org/
```
Comment (ngocvt-0484) [inline config/settings.yml]: please read it
https://docs.google.com/document/d/104Csp4-vamVos5DEbi372nLBEfmqhj7h1ENYO8ebjCo/edit
and add endline same as No 12

**Comment by meimei0111** [comment]
merged
Vanvtt said :D

**Pull request merged**